builtin/mainmenu/tab_settings.lua

```diff
@@ -64,13 +64,13 @@ local dd_antialiasing_labels = {
 	fgettext("None"),
 	fgettext("2x"),
 	fgettext("4x"),
-	fgettext("8x"),
+	fgettext("8x")
```

Comment:
Don't remove the comma, its bad style. If we want to add additional elements, we don't want to modify this line.